lab08-kaylee/README.md

```diff
@@ -0,0 +1,44 @@
+## Lab-08: Single-Resource API
+
+## About This Project
+
+* Build an HTTP server using the 'http' Node module.
+* Create an Object constructor used to produce instances of a resource.
+* Create a Router constructor to manage GET, POST, PUT and DELETE requests to the server.
+
+## Project Dependencies
+
+* chai, chai-http, debug and uuid
+* To install:
```

Comment:
technically, to install, it's just npm install once they pull down your repo if your dependencies/dev dependencies are saved in your package.json :)